test/unit/app/utils/ussd/kenya/test_pin_state.py

```diff
@@ -62,49 +62,54 @@ def test_kenya_state_machine(test_client, init_database, user_factory, session_f
     assert state_machine.state == expected
 
 
-@pytest.mark.parametrize("session_factory, user_factory, user_input, expected, before_failed_pin_attempts, after_failed_pin_attempts",
- [
-     # send token pin auth combinations
-     (send_token_pin_authorization_state, pin_blocked_user, "1212", "exit_pin_blocked", 3, 3),
-     (send_token_pin_authorization_state, standard_user, "1212",
-      "send_token_pin_authorization", 1, 2),
-     (send_token_pin_authorization_state, standard_user, "0000",
-      "send_token_confirmation", 1, 0),
-     (send_token_pin_authorization_state, standard_user, "1212",
-      "exit_pin_blocked", 2, 3),
-     (send_token_pin_authorization_state, standard_user, "0000",
-      "send_token_confirmation", 2, 0),
-     # balance inquiry pin auth combinations
-     (balance_inquiry_pin_authorization_state, pin_blocked_user, "1212", "exit_pin_blocked", 3, 3),
-     (balance_inquiry_pin_authorization_state, standard_user, "1212",
-      "balance_inquiry_pin_authorization", 1, 2),
-     (balance_inquiry_pin_authorization_state, standard_user, "0000",
-      "complete", 1, 0),
-     (balance_inquiry_pin_authorization_state, standard_user, "1212",
-      "exit_pin_blocked", 2, 3),
-     (balance_inquiry_pin_authorization_state, standard_user, "0000",
-      "complete", 2, 0),
-     # exchange token pin auth combinations
-     (exchange_token_pin_authorization_state, pin_blocked_user, "1111", "exit_pin_blocked", 3, 3),
-     (exchange_token_pin_authorization_state, standard_user, "1212",
-      "exchange_token_pin_authorization", 1, 2),
-     (exchange_token_pin_authorization_state, standard_user, "0000",
-      "exchange_token_confirmation", 1, 0),
-     (exchange_token_pin_authorization_state, standard_user, "1212",
-      "exit_pin_blocked", 2, 3),
-     (exchange_token_pin_authorization_state, standard_user, "0000",
-      "exchange_token_confirmation", 2, 0),
-     # exchange rate pin auth combinations
-     (exchange_rate_pin_authorization_state, pin_blocked_user, "1111", "exit_pin_blocked", 3, 3),
-     (exchange_rate_pin_authorization_state, standard_user, "1212",
-      "exchange_rate_pin_authorization", 1, 2),
-     (exchange_rate_pin_authorization_state, standard_user, "0000",
-      "complete", 1, 0),
-     (exchange_rate_pin_authorization_state, standard_user, "1212",
-      "exit_pin_blocked", 2, 3),
-     (exchange_rate_pin_authorization_state, standard_user, "0000",
-      "complete", 2, 0),
- ])
+@pytest.mark.parametrize("session_factory, user_factory, user_input, expected, before_failed_pin_attempts, "
+                         "after_failed_pin_attempts",
```

Comment:
What do you mean when you say keep the old format, do you mean the indentation ?

Reply:
Yes, exactly!

Reply:
In the interest of standardization maybe we should come up with a consistent styling toolkit for the code base, so that any dev can configure their IDE to said styling guide.
Because in this particular instance this was a change PyCharm auto implemented.

Reply:
I totally agree. We should definitely try to settle on some PyLint settings sooner than later.